app/js/components/ReactReduxSelect.jsx

```diff
@@ -0,0 +1,36 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+import React from 'react';
+import Select from 'react-select-plus';
+import style from '!style-loader!css-loader!react-select-plus/dist/react-select-plus.css'; // eslint-disable-line
+
+
+class ReactReduxSelect extends React.Component {
```

Comment:
Well, the original select is `react-select`, and the point of putting this in a component is to make it additionally `Redux` compatible (if I understand correctly why what I did worked, which is sort of a toss up).  The original kit works immediately with React so I feel like my containerization of it to work with Redux makes this name make sense.  If I didn't either a) stick it in a container like this or b) use some super ugly `React.DOM` code, the `Redux` state tree wiped the value in the select text box as soon as it was entered, which is definitely not the optimal behavior.  What do you think?